src/plugins/circular_view/src/CircularItems.cpp

```diff
@@ -456,14 +457,15 @@ bool CircularAnnotationLabel::tryPushCounterclockwise() {
     const int engagedIndex = ra->engagedLabelPositionToLabel.key(this, -1);
     SAFE_POINT(-1 != engagedIndex, "Unexpected label position index", false);
     const int numberOfEngagedIndex = suitableLabelPositionIndexes.indexOf(engagedIndex);
-    SAFE_POINT(-1 != numberOfEngagedIndex, "Unexpected label position index", false);
 
-    if (0 == numberOfEngagedIndex) {
+    if (0 == numberOfEngagedIndex
```

Comment:
Same here: do not use yoda style (compiler already produces compile-time error if you will have an assignment instead of comparison in this case) + add a comment that describes what is going on here.